website/tests/test_website_models.py

```diff
@@ -0,0 +1,191 @@
+import pytest
+import time
+from distutils.version import LooseVersion
+from django.utils import timezone
+from ..models import Profile, WebsiteSection, NewsPost, \
+    Publication, CarouselImage, DocumentationLink
+
+
+def test_string_representation(random_user):
+    entry = WebsiteSection(title="My entry title")
+    assert str(entry) == entry.title
```

Comment:
You override __str__() function on each model so I test we get the desired value that you define. So I think it's fine here.